index.js

```diff
@@ -3,14 +3,13 @@
 var fs = require('fs');
 var path = require('path');
 var log = require('gulplog');
+var fancyLog = require('fancy-log');
```

Comment:
`toConsole` can be updated to remove previous listeners if called a 2nd time.  You can then use it at the beginning of this file and call it again after you normalize the config.